src/app/components/pages/Question.tsx

```diff
@@ -76,7 +76,7 @@ export const Question = withRouter(({questionIdOverride, match, location, previe
         const isFastTrack = doc && doc.type === DOCUMENT_TYPE.FAST_TRACK_QUESTION;
 
         return <GameboardContext.Provider value={navigation.currentGameboard}>
-            <Container className={`${doc.subjectId || ""}`}>
+            <Container className={classNames(`${doc.subjectId || ""}`, "no-shadow")}>
```

Comment:
The `classNames` helper lets you not have to worry about whether something is a string or undefined or whatever; it will just build a valid value for you. So if you're moving to using it, we don't need the nasty workaround that was there:
```suggestion
            <Container className={classNames(doc.subjectId, "no-shadow")}>
```